system-tests/run.py

```diff
@@ -141,16 +141,18 @@ def rule_binding_apply_test(namespace="kubecop-test"):
     except Exception as e:
         print("Exception occured: %s" % e)
         return 1
+    
+    return 0
 
     return 0
 
 
 test_cases = [
-    (basic_alert_test, "Basic alert test"),
     (load_10k_alerts_no_memory.load_10k_alerts_no_memory_leak, "Load 10k alerts no memory leak test"),
     (creation_app_profile_memory_leak.install_app_no_application_profile_no_leak, "Install app no application profile no leak test"),
     (kill_in_the_middle.kill_process_in_the_middle, "Kill process in the middle test"),
     (rule_binding_apply_test, "Rule binding apply test"),
+    (basic_alert_test, "Basic alert test"),
```

Comment:
I deliberately put this as the first test. If it fails there is no point running the others. It is the first one to check (smoke test)